lib/bulk_sql_delete.rb

```diff
@@ -0,0 +1,62 @@
+class BulkSqlDelete
+  attr_accessor :connection
+
+  def initialize(connection = ActiveRecord::Base.connection)
+    self.connection = connection
+  end
```

Comment:
I don't think we need the initializer, we can just get the connection from the pool inside `delete_in_batches`

Reply:
This connection becomes important if we ever decide to have multiple databases but I can remove it for now. This code is based on similar code I wrote before and that code was supporting multiple dbs.